core/tests/karma.conf.ts

```diff
@@ -31,7 +31,7 @@ module.exports = function(config) {
       'core/templates/dev/head/**/*_directive.html',
       'core/templates/dev/head/**/*.directive.html',
       'core/templates/dev/head/**/*.template.html',
-      'core/templates/dev/head/AppInit.ts',
+      'core/templates/dev/head/pages/about-page/about-page.module.ts',
```

Comment:
I have removed `AppInit.ts` since it wasn't needed anywhere except here. I think we can use any module that we have, but I picked about page because it is first alphabetically.

Reply:
Perhaps it would be a good idea to note this as a comment in the file? Otherwise it seems weird to single it out.